swagger/swagger.yaml

```diff
@@ -1370,6 +1370,27 @@ paths:
             security:
                 - oauth2: []
             summary: Sign out
+        get:
+            description: List all users with the groups they belong to
```

Comment:
I would try to keep our vocabulary to a minimum and avoid synonyms. So I would stick to `Get all` instead of `list`